packages/sui-studio/package.json

```diff
@@ -45,7 +45,8 @@
     "raw-loader": "4.0.2",
     "react": "17",
     "react-dom": "17",
-    "react-docgen": "5.3.1"
+    "react-docgen": "5.3.1",
+    "react-test-renderer": "17.0.1"
```

Comment:
"react-docgen": "5",
 "react-test-renderer": "17"

Reply:
No usamos major para dependencias externas (normalmente).

Reply:
I'm going to resolve this conversation due the external dependencies should be defined without majors. Thank you! 🙌